Userland/Utilities/realsize.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <AK/Optional.h>
+#include <AK/StringView.h>
+#include <AK/Types.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    TRY(Core::System::pledge("stdio rpath"));
+
+    StringView argument;
+
+    Core::ArgsParser args_parser;
+    args_parser.set_general_help(
+        "Show the 'real' size of a number with a suffix (possibly).");
+    args_parser.add_positional_argument(argument, "Number with possibly a suffix", "number");
+    args_parser.parse(arguments);
+
+    if (argument.is_null() || argument.is_empty())
+        return Error::from_string_literal("Invalid value");
+
+    unsigned suffix_multiplier = 1;
+    auto suffix = argument[argument.length() - 1];
+
+    switch (suffix) {
+    case 'k':
```

Comment:
As discussed on Discord, lowercase is usually reserved for `*bits` instead of `*bytes`. We should most likely just have `KiB` and `KB` as separate cases. `K` behavior remains to be decided, probably with two separate mutually exclusive but required flags.

Reply:
As far as I know, just `K` means `KiB` most places (😔)